src/api/app/models/bs_request.rb

```
@@ -114,7 +114,8 @@ def assign_number
     BsRequest.transaction do
       request_counter = BsRequestCounter.lock(true).first_or_create
       self.number = request_counter.counter
-      request_counter.increment!(:counter)
+      request_counter.increment(:counter)
+      request_counter.save
```

Might be better to call save! here. Otherwise this might fail silently